fastlane/Fastfile

```diff
@@ -915,6 +915,15 @@ platform :ios do
     ios_localize_project
   end
 
+  desc 'Lint the `.strings` files'
+  lane :lint_localizations do
+    ios_lint_localizations(
+      input_dir: RESOURCES_FOLDER,
+      allow_retry: true,
+      check_duplicate_keys: false # skip for 9.3, which has an issue in the en file
```

Comment:
This should be removed before the next code freeze

For future reference: `git revert 88c806b2af`